### DISCUSSION TITLE
사용자의 접근성을 높이기 위해 고려 해볼 만한 사항들

### DISCUSSION THREAD

**Discussion by jlee38266** [OmokNoonE/OnionHotSayYo-backend#10] [의견 교환]
처음에 저희가 계획 했었던 OmokLens는 해커톤이나 포럼과 같은 정보를 제공해주는 웹사이트를 생각했지만, 피드백을 통해 다른 웹사이트의 데이터베이스를 끌어다 오는 것은 실제 서비스를 제공하는 웹사이트 측면에서는 임팩트가 낮고, 타사의 데이터베이스 의존도가 높다는 문제점, 만약 관련 대회 또는 포럼 관계자 또는 트랜드를 파악할 수 있는 프로젝트를 등록할 수 있도록 유도할만한 요소가 고민 되지 않을 것 같다는 피드백을 받았습니다. 우리가 만들 사이트를 정말로 완벽하게 만들 필요는 없지만, 실제 서비스를 배포했을 때, 이러한 부분이 고려되어야 의미 있는 프로젝트가 될 수 있다는 피드백을 받았습니다.

현재 저희가 제작하고 있는 OmokLens는 종합 커뮤니티의 성격을 띄고면 좋지 않을까 생각합니다. 다만 저희 웹사이트가 활성화 되기 위해선 사용자가 적극적으로 참여 하고, 자신의 생각 또는 진행하고 있는 특정 이벤트를 공유를 할 때 의미가 있는 데이터가 쌓이고 이를 통해서 선순환이 반복됩니다. 이러한 문제들을 함께 고민하면 좋지 않을까 생각합니다.

**Comment by jlee38266** [comment]
저는 개인적으로 대학생과 같은 프로젝트 또는 정보를 접근하는데 미숙한 초보자를 타켓으로 삼고 이들에게 웹사이트를 사용해야 할 동기부여가 필요하다고 생각합니다.

예를 들어
- 인터렉티브 투어: 사이트의 주요 기능과 사용 방법을 안내하는 튜토리얼
- 멘토링 프로그램: 경험이 있는 학생 또는 전문가가 멘토로 등록되어 사람들에게 정보를 공유할 수 있는 장을 만들기
- 커스텀 프로젝트 추천: 방향성이 없는 사용자에게 시도 할 수 있는 프로젝트를 추천하는 알고리즘
- 성공 사례 하이라이트: 프로젝트가 성공적으로 완료한 사용자의 사례 소개
- 사용자 피드백 반영: 사용자의 피드백을 정기적으로 수집할 수 있는 공간 확보 및 사이트에 반영
- 보상 시스템: 정보를 등록하는 사용자에게 포인트, 뱃지, 인증서와 같은 금전적 이외의 보상
- 사용자 프로필 커스텀화: 사용자가 자신의 관심사나 기록을 하면서 하나의 포트폴리오 성격을 띌 수 있게 유도
- 대회 참여 시 가이드 라인 제공: 특정 대회에 관심이 있는 사용자에게 해당 대회에 나가기 위해서 어떤 준비가 필요 한지에 대한 가이드라인 제공

이런 항목들을 예시로 사용자의 측면에서 매력적으로 느낄 수 있는 부분이 고려 되면 좋을 것 같습니다.

**Comment by jlee38266** [comment]
크롤링에 대한 부분도 피드백을 받았습니다. 만약 단순히 다른 웹사이트의 데이터를 가져 오는 것은 평가를 받았을 때 (면접과 같은) 이 기술을 왜 적용했는지 적절한 대답을 하기 힘들 것 같다는 피드백을 받았습니다 (또한 타 웹사이트의 데이터베이스를 가져 오는 것은 물론 교육 목적이긴 하지만 서비스 배포라는 관점까지 고려한다면 법적인 문제가 발생할 문제가 많다는 지적을 받았습니다). 차라리 크롤링 기술을 통해 우리가 원치 않는 데이터들을 찾아내서 관리할 수 있는 방향으로 적용해 보는 것도 좋을 것 같다는 피드백을 받았습니다. 

기술적인 부분을 보여주는 것은 평가자 입장에서는 크게 매력적이지 않다는 (결코 무시 되어서는 아니 되지만 기술적인 부분은 이미 많은 자료가 배포 되어 있고 순수한 기술적인 부분의 경쟁은 현재 수준에서는 경쟁력이 될 수 없다는 평가) 평가입니다. 사용자 경험 측면의 고민을 조금이라도 해보는 것을 권장 받았습니다 (개연성 + @).

**Comment by ms1011** [comment]
저도 저희 사이트의 방향성을 `커뮤니티`의 형태로 띄고 있는 것이 좋을 것 같습니다.
크롤링은 특정사이트가 망하면 저희 서비스도 더이상 운영할 수 없다는 부분과 저작권에서 문제가 생길 것으로 보입니다.

`커뮤니티`라는 부분에서 개발자만을 위한 커뮤니티는 너무 작은 부분일 수 있어
포트폴리오로 `프로젝트`를 만들어야하는 직종이나 업계를 타겟으로 하면 좋을 것 같습니다.
이렇게 할 경우 추후에 `오목허브`의 확장성을 고려해 오목허브도 개발자뿐 아닌 프로젝트를 필요로하는 모든이에게 서비스를 제공할 수 있을 것으로 보입니다.

커뮤니티 사이트의 특징으로 가져갈 수 있는 부분을 예시로 드셨던 부분에 의견을 달자면
- 인터렉티브 투어: 제가 생각했을 때는 꼭 필요한 기능인지 모르겠습니다. 추후에는 넣을 수 있을 것 같습니다.
- 멘토링 프로그램: 전문가를 멘토로 하기에는 영업이 필요한 부분이라 [`김과외`](https://kimstudy.com/) 사이트와 같이 멘토 멘티를 선정하는 부분도 좋아보입니다.
- 커스텀 프로젝트 추천: 이 부분은 추후에 오목허브와 연계하여 사용할 수 있을것 같습니다
- 성공 사례 하이라이트: 이 부분도 위와 같습니다
- 사용자 피드백 반영: QnA, FAQ를 통해 피드백을 받으면 좋을 것 같습니다.
- 보상 시스템: 오목허브의 뱃지 or 업적 시스템과 연동하면 더욱 좋을 것 같습니다
- 사용자 프로필 커스텀화: 이 부분은 오목허브의 부분으로 보입니다.
- 대회 참여 시 가이드 라인 제공: 공모전 부분을 버린다면 필요 없을 것 같습니다

**Comment by CJC0512** [comment]
1. 블로그 => 노션
2. 김과외
3. GPS기반 커뮤니티 서비스
4. ~~일반 개발자 커뮤니티~~



예린: 김과외, 블로그

민석: GPS, 블로그

종찬: 블로그, GPS

현근: 블로그, GPS

재원: 블로그, GPS



짬뽕

1. 블로그 + 김과외
   1. 멘티: 회고록
   2. 멘토: 강의자료 정리
   3. 블로그 => 멘토의 강의 블로그
2. ~~김과외 + GPS~~
3. 블로그 + GPS (=지역 커뮤니티)
   1. 지역블로그(대학, 초중고등학교 커뮤니티, 회사 커뮤니티,  지하철역 커뮤니티(가산디지털단지), 랜드마크 커뮤니티)

**Comment by CJC0512** [comment]
1. ~~git-hub 사용 가이드 커뮤니티 (+ jira, slack 연동 등 발전 가능성 有)~~
2. ~~오목허브~~
3. 블로그(자동완성 + 템플릿 공유)
   1. 육아수첩
   2. ~~애완동물 블로그~~

4. ~~지속가능한 생활 가이드 웹 사이트 (재활용, 분리수거 등)~~

5. ~~자취 커뮤니티 웹 사이트 (통합의 개념, 기본 개념 제공 사이트)~~

6. 문화 교류 웹 사이트 (해외 지역정보 공유 사이트, flat mate?, 외국인 친구 사귀기, 광고, 양측 모두가 사용하기 쉬운 웹, 구글 로그인)
